phoenix-core/src/main/protobuf/PTable.proto

```diff
@@ -111,6 +111,9 @@ message PTable {
   optional bool viewModifiedPhoenixTTL = 44;
   optional int64 lastDDLTimestamp = 45;
   optional bool changeDetectionEnabled = 46;
+  optional bytes physicalTableNameBytes = 47;
+  optional bool isModifiable = 48;
```

Comment:
where is isModifiable set?

Reply:
It will be set later as part of another change. Since I was changing this part, I added this one as well